src/main/cpp/src/CastStringJni.cpp

```diff
@@ -279,7 +279,8 @@ Java_com_nvidia_spark_rapids_jni_CastStrings_parseTimestampStrings(JNIEnv* env,
                                                                    bool is_default_timezone_dst,
                                                                    long default_epoch_day,
                                                                    jlong timezone_info_column,
-                                                                   jlong transitions_table)
+                                                                   jlong transitions_table,
+                                                                   jboolean is_spark_320)
```

Comment:
Since the changes target Spark 320 to 355, should this?
```suggestion
                                                                   jboolean is_from_spark_320)
```

Reply:
If this applies to Spark 320 to 355 then should it be `is_from_spark_320`?